nvtabular/workflow.py

```diff
@@ -695,15 +697,16 @@ def apply(
         out_files_per_proc : integer
             number of files to create (per process) after
             shuffling the data
+        num_io_threads : integer
+            Number of IO threads to use for writing the output dataset.
+            For `0` (default), no dedicated IO threads will be used.
         """
 
-        # Deal with single-gpu compatibility
-        nsplits = kwargs.get("nsplits", None)
+        # Deal with 0.1.1 compatibility
         if nsplits:
             warnings.warn("nsplits is deprecated. Use out_files_per_proc")
```

Comment:
If we're going to keep this warning in place (instead of just removing the old arguments), can we use a deprecation warning instead? 
```suggestion
            warnings.warn("nsplits is deprecated. Use out_files_per_proc", DeprecationWarning)
```

Reply:
you've convinced me - lets just yank the old args :)